js/services/MeteorService.ts

```diff
@@ -1,28 +1,30 @@
-const requestUtils = require('../clients/MeteorClient')
-const MeteorPerDateDto = require('../dtos/MeteorPerDateDto')
-const meteorMapper = require('../mappers/MeteorMapper')
+import { getMeteorsWithinPeriod } from '../clients/MeteorClient'
+import { MeteorDto } from '../dtos/MeteorDto'
+import { MeteorPerDateDto } from '../dtos/MeteorPerDateDto'
+import { buildMeteorEntity } from '../mappers/MeteorMapper'
+import { MeteorEntity, MeteorsDataItem, NearEarthObjects } from '../models/MeteorModels'
 
-const getMeteorsData = async (meteor) => {
-    const meteorsDataResponse = await requestUtils.getMeteorsWithinPeriod(meteor.startDate, meteor.endDate)
-    const nearEarthObjects = meteorsDataResponse.data.near_earth_objects
+export const getMeteorsData = async (meteor: MeteorDto) => {
+    const meteorsDataResponse = await getMeteorsWithinPeriod(meteor.startDate, meteor.endDate)
+    const nearEarthObjects = meteorsDataResponse.data.near_earth_objects as NearEarthObjects
```

Comment:
Is it good/bad practise? 
Line 8: I get some complex object from request (do not have specific type for it). 
Line 9: Specify object type

Reply:
Using 'as' means manually typing, it's not a good idea. 'meteorsDataResponse' should have type/interface with NearEarthObjects.
For instance:
`type MeteorsData {
near_earth_objects: NearEarthObjects
}`
if 'meteorsDataResponse' has any, you will type 'await getMeteorsWithinPeriod(meteor.startDate, meteor.endDate)', 
for example, create generic.
In this case, you can also destructure 'meteorsDataResponse ': 
`const { data } =  await getMeteorsWithinPeriod(meteor.startDate, meteor.endDate)` and `const nearEarthObjects = data.near_earth_objects as NearEarthObjects`